signac/testing.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2019 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+from __future__ import print_function
```

Comment:
This isn't needed since we are dropping Python 2.7 support and are not using the print function here anyways.